src/Raven.Client/Documents/Operations/ETL/AI/AzureOpenAiSettings.cs

```diff
@@ -0,0 +1,54 @@
+﻿#nullable enable
+using System;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.ETL.AI;
+
+public sealed class AzureOpenAiSettings : OpenAiBaseSettings
+{
+    public AzureOpenAiSettings(string apiKey, string endpoint, string model, string deploymentName, int? dimensions = null) : base(apiKey, endpoint, model)
+    {
+        DeploymentName = deploymentName;
+        Dimensions = dimensions;
+    }
+    
+    public AzureOpenAiSettings()
```

Comment:
there is few warnings in all those ctors that @Lwiel pushed due to serialization issues, @ArieSLV please take a look at those

Reply:
Done